src/background/messaging/rpc-methods/sign-psbt.ts

```diff
@@ -29,8 +30,8 @@ function validatePsbt(hex: string) {
 }
 
 export async function rpcSignPsbt(message: SignPsbtRequest, port: chrome.runtime.Port) {
-  if (!validateRpcSignPsbtParams(message.params)) {
-    const errors = getRpcSignPsbtParamErrors(message.params);
+  if (!message.params || !validateRpcParams(message.params, rpcSignPsbtValidator)) {
```

Comment:
Could we refactor `validateRpcParams` to import `rpcSignPsbtValidator` itself rather than having to pass it as an argument?